<launch>

	<node name="streamer" pkg="thermalvis" type="streamer">
		<param name="source" value="directory" />
		<!-- <param name="directory" value="~\Documents\GitHub\thermalvis\media\sample_video\optris" /> -->
		<param name="directory" value="C:\Users\Stephen\Documents\Data\optris\test_seq_001" />
		<param name="outputFolder" value="~\Documents\Data\optris\output" />
		<param name="writeImages" value="false" />
		
		<!-- <param name="tempGrad" value="100.0" /> -->
		<!-- <param name="tempIntercept" value="0.0" /> -->
		
		<!-- <param name="minTemperature" value="25" /> -->
		<!-- <param name="maxTemperature" value="35" /> -->
		
		<param name="loopMode" value="true" />
		<!-- <param name="autoTemperature" value="true" /> -->
		
	</node>

	
	<node name="flow" pkg="thermalvis" type="flow" output="screen">
		
		<param name="debugMode" value="true" />
		<param name="showTrackHistory" value="true" />
		<param name="autoTrackManagement" value="false" />
		<!-- <param name="verboseMode" value="true" /> -->
		
		<!-- <param name="detector_1" value="FAST" /> -->
		<!-- <param name="minFeatures" value="100" /> -->
		<!-- <param name="sensitivity_1" value="0.05" /> -->
		<!-- <param name="flowThreshold" value="0.001" /> -->
		
	</node>
	
</launch>
